ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb

```diff
@@ -13,6 +13,15 @@
     "Let's start by importing required libraries and initializing session and other variables used in this notebook. By default, the notebook uses the default Amazon S3 bucket in the same AWS Region you use to run this notebook. If you want to use a different S3 bucket, make sure it is in the same AWS Region you use to complete this tutorial, and specify the bucket name for `bucket`. "
```

Comment:
<h2>"Launch jobs (optional)"</h2><p> </p><p>"Now that you've created your manifests, you're ready to launch your adjustment labeling job. Use this template for launching labeling jobs via <a href="https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sagemaker.html" target="_blank">Boto3</a>. In order to access the labeling job, make sure you followed the steps to create a private work team."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2134/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>